textile/chat-features.textile

```diff
@@ -172,12 +173,25 @@ h2(#room-configuration). Room Configuration
-** @(CHA-RC1d)@ @[Testable]@ If a room is requested to be released, it is immediately removed from the SDKs room map - meaning subsequent calls to get an instance of a room for the same ID shall produce a new object.
-*** @(CHA-RC1d1)@ @[Testable]@ Any attach operations on the new room instance must not commence until the release operation fully completes (including any Realtime channel releasing). This is to avoid resource leakage and accidentally removing the underlying channels during transition.
-** @(CHA-RC1e)@ @[Testable]@ Pursuant to @CHA-RC1d@, the room releasing lifecycle operations (see "@CHA-RL3@"#CHA-RL3) shall occur @AFTER@ the room is removed from the room map, but shall complete before the call to release the room completes.
+** @(CHA-RC1a)@ This specification point has been removed. It was superseded by CHA-RC1f.
+** @(CHA-RC1f)@ @[Testable]@ Requesting a room from the Chat Client will return a future, that eventually resolves to an instance of a room with the provided id and options.
+*** @(CHA-RC1f1)@ @[Testable]@ If the room id exists in the room map, but the room has been requested with different options, then an @ErrorInfo@ with code @40000@ is thrown.
+*** @(CHA-RC1f2)@ @[Testable]@ If the room id exists in the room map, and it is requested with the same options, then the same instance of the room will be reused.
+*** @(CHA-RC1f3)@ @[Testable]@ If no @CHA-RC1g@ release operation is in progress, a new room instance will be created, added to the room map and returned as the future value.
+*** @(CHA-RC1f4)@ @[Testable]@ If a @CHA-RC1g@ release operation is in progress, the entry will be added to the room map, but the new room instance will not be created until the release operation has resolved. The future will then subsequently resolve with the new room value.
+*** @(CHA-RC1f5)@ In relation to @CHA-RC1f4@, we recommend storing a future in the room map.
+*** @(CHA-RC1f6)@ It should be noted that the "get" operations in this section are interruptible by a @CHA-RC1f@ release call.
+** @(CHA-RC1b)@ This specification point has been removed, it was superseded by @CHA-RC1f2@
+** @(CHA-RC1c)@ This specification point has been removed, it was superseded by @CHA-RC1f3@
+** @(CHA-RC1d)@ This specification point has been removed.
+*** @(CHA-RC1d1)@ This specification point has been removed.
+** @(CHA-RC1e)@ This specification point has been removed.
+** @(CHA-RC1g)@ A room may be @released@, which causes it to be disposed of and eligible for garbage collection.
+*** @(CHA-RC1g1)@ The release operation returns a future that will resolve when the operation completes. 
+*** @(CHA-RC1g2)@ @[Testable]@ If the room does not exist in the room map, and no release operation is in progress, this operation is no-op.
+*** @(CHA-RC1g3)@ @[Testable]@ If the room does not existing in the room map, and a release operation is already in progress, then the associated future will be returned.
```

Comment:
*exist*

Reply:
I like the idea that a room does "existing" 😆 

Fixed!